src/Cli/dotnet/commands/dotnet-test/IPC/Models/TestResultMessages.cs

```diff
@@ -1,13 +1,13 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Why change the license?

Reply:
This was done automatically upon saving. I removed this change.